buddyboss/buddyboss-platform
# PROD-678 View page of the Elementor saved template not working - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3194  .

### How to test the changes in this Pull Request:

1. Make sure the News feed has been set as home page.
2. Go to Templates > Saved Templates
3. Click on the view for any saved templates
4. See it workds

### Proof Screenshots or Video

https://share.vidyard.com/watch/x7xJERGzLaU17oA9ZKEfRM?

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
